server/src/main/java/com/flipkart/varadhi/web/v1/admin/SubscriptionHandlers.java

```diff
@@ -38,34 +39,34 @@ public List<RouteDefinition> get() {
                 List.of(
                         RouteDefinition
                                 .get("ListSubscriptions", "")
-                                .authorize(SUBSCRIPTION_LIST, "{project}")
+                                .authorize(SUBSCRIPTION_LIST)
                                 .build(this::getHierarchy, this::list),
                         RouteDefinition
                                 .get("GetSubscription", "/:subscription")
-                                .authorize(SUBSCRIPTION_GET, "{project}/{subscription}")
+                                .authorize(SUBSCRIPTION_GET)
                                 .build(this::getHierarchy, this::get),
                         RouteDefinition
                                 .post("CreateSubscription", "")
                                 .hasBody()
                                 .bodyParser(this::setSubscription)
-                                .authorize(SUBSCRIPTION_CREATE, "{project}")
+                                .authorize(SUBSCRIPTION_CREATE)
```

Comment:
this will require larger changes either to pass AuthorizationProvider to SubscriptionHandlers or to make changes to provider contract to handle multiple actions.

Reply:
yes, I'll raise it as a separate PR